.github/workflows/build.yml

```diff
@@ -14,14 +14,15 @@ jobs:
       fail-fast: false
       matrix:
         group:
-          - "unit"
+          - "unit-core"
+          - "unit-storage"
           - "tutorial"
           - "guide"
           - "biology"
           - "faq"
           - "storage"
           - "cli"
-    timeout-minutes: 11
+    timeout-minutes: 10
```

Comment:
This should be back to 7 min.

Reply:
We have to invest to get us strictly under 4 min in the upcoming days and weeks.